state/core_access.go

```diff
@@ -184,9 +184,9 @@ func (ca *CoreAccessor) cancelCtx() {
 	ca.cancel = nil
 }
 
-// SubmitPayForBlob builds, signs, and synchronously submits a MsgPayForBlob with additional options defined
-// in `TxConfig`. It blocks until the transaction is committed and returns the TxResponse.
-// The user can specify additional options that can bee applied to the Tx.
+// SubmitPayForBlob builds, signs, and synchronously submits a MsgPayForBlob with additional
+// options defined in `TxConfig`. It blocks until the transaction is committed and returns the
+// TxResponse. The user can specify additional options that can bee applied to the Tx.
```

Comment:
lets try to avoid unnecessary changes in this PR.

(I know this lint issue is annoying and we have to fix it in a separate PR)

Reply:
No, unnecessary nit. I got asked to make the linter pass. I'm not going to continue subjecting to back and forths in a PR review

Reply:
It's valid to ask not to make changes in irrelevant pkgs. The CI linter would pass without those, and the mismatch between ci and local has to be fixed anyway. I am not requesting changes with this comment

Reply:
I fine to live that as it is, as an exception